tracer/src/Datadog.Trace/Debugger/ExceptionAutoInstrumentation/TestExceptionNormalizer.cs

```diff
@@ -0,0 +1,36 @@
+// <copyright file="TestExceptionNormalizer.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+#nullable enable
+namespace Datadog.Trace.Debugger.ExceptionAutoInstrumentation
+{
+    /// <summary>
+    /// Important: Should only be used in testing. Not thread-safe.
```

Comment:
Can we put this class in `Datadog.Trace.Tests.Debugger`?

Reply:
Unfortunately, no. It's uses `ReadOnlySpan` that is weaved as part of `Datadog.Trace`. If I try to move it elsewhere, I'll have to adjust the implementations of `MemoryExtesions`. Does that make sense?